src/EditorFeatures/CSharpTest/UseIndexOrRangeOperator/UseRangeOperatorTests.cs

```diff
@@ -221,6 +219,94 @@ void Goo(string s, string t)
     {
         var v = t.Substring(s[1..^1][0], t.Length - s[1..^1][0]);
     }
+}", parseOptions: s_parseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseRangeOperator)]
+        public async Task TestWithTypeWithActualSliceMethod1()
```

Comment:
💡 It would be nice to have a test for slicing on the left side of an assignment. This is valid when the `Slice` method returns a `ref` type.

Reply:
sure.

Reply:
Added.